integration_tests/tests/apply/prison_applications.cy.ts

```diff
@@ -0,0 +1,28 @@
+//  Feature: referrer views prison applications
+//    So that I can see recently created applications for my prison
+//    So that I can manage applications
+//
+//  Scenario: viewing the prison applications page as a referrer
+//      Given I am logged in as a prison bail referrer
+//      And there are allocated applications for my prison in the database
```

Comment:
strictly speaking this shouldn't be here yet

Reply:
I'll amend the scenario 👍🏾